docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,122 @@
+# Installing Kubeapps on BKPR
+
+This guide documents the installation of [Kubeapps](https://kubeapps.com/) to your Kubernetes Production Runtime (BKPR) cluster. You will first install the [Helm](https://www.helm.sh/) package manager to the cluster, followed by the installation of Kubeapps using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps).
+
+![Kubeapps Application Catalog](images/kubeapps-app-catalog.png)
+
+The guide assumes you have a Kubernetes cluster with BKPR already installed. The [BKPR installation guide](install.md) documents the process of installing the BKPR client, followed by installing the BKPR server side components to the cluster.
+
+## Step 1: Install the Helm client
+
+Follow the instructions in [Helm installation guide](https://docs.helm.sh/using_helm/#installing-the-helm-client) to install the latest release of the Helm binary to your machine.
+
+## Step 2: Install the Tiller server
+
+Tiller is the server-side component of Helm that runs inside your Kubernetes cluster. To install the Tiller server to the cluster you should use the `helm` client installed by the previous step. However, before we do so, for RBAC enabled clusters we need to create a service account for the Tiller server.
+
+For the purposes of this guide we'll create a Kubernetes service account with super-user cluster access (`cluster-admin`). Check out the guide on [Tiller and Role-Based Access Control](https://github.com/helm/helm/blob/master/docs/rbac.md) to learn about restricting the scope of Tiller in the cluster.
```

Comment:
Iirc, this leaves the tiller endpoint exposed within the cluster, with super-user access and with no authentication.  I don't think we can suggest this (at least without putting the "not fit for production" warning back in).

The [kubeapps tiller/TLS docs](https://github.com/kubeapps/kubeapps/blob/master/docs/user/securing-kubeapps.md) look good.  I think we should point to them (or use their example commands) during both the tiller and kubeapps install.  

For bonus points, it would be awesome if we can use cert-manager Certificate objects to generate the self-signed CA+user certs - but I'm not sure how hard that is given the existing `helm install` command line options (wants local paths rather than existing Secrets).  This might be too hard for now however.

Note this would be different if this was a helm-only doc (not kubeapps), since then we'd need to consider other tiller clients beyond kubeapps (and thus talk about per-user tiller installs, per-user certs, etc).

Reply:
I have whacked the section from the Kubeapps guide.